uaclient/messages.py

```diff
@@ -73,6 +73,9 @@ class TxtColor:
 )
 USN_FIXED = "{issue} is addressed."
 CVE_FIXED = "{issue} is resolved."
+CVE_FIXED_BY_LIVEPATCH = (
+    "{issue} is resolved by livepatch patch version: {version}."
```

Comment:
Let's add the green check in front of this. (The above strings USN_FIXED and CVE_FIXED are misleading in not having the check)
```suggestion
    OKGREEN_CHECK + " {issue} is resolved by livepatch patch version: {version}."
```